src/test/resources/changesets/34_add_client_scope_mappers.yml

```diff
@@ -0,0 +1,37 @@
+id: add-client-scope-mappers
+author: janunld
+realm: integ-test
+changes:
+  - addClientScope:
+      name: testMappers
+  - addClientScopeAudienceMapper:
+      clientId: testMappers
```

Comment:
this should be clientScopeName if i'm not mistaken, same for the following addClientScope mapper commands